lib/jira/commands/install.rb

```diff
@@ -15,35 +15,52 @@ class Install < Base
 
       def run
         io.say('Please enter your JIRA information.')
-        inifile[:global] = params
+        inifile[:global] = base_params
+        inifile.write # Do this now because cookie authentication uses api calls
+
+        inifile.delete_section("cookie") if inifile.has_section?("cookie")
+        case authentication
+        when "basic"
+          inifile[:global][:password] = password
+        when "token"
+          inifile[:global][:token] = token
```

Comment:
Would be nice if we can exchange username / password for a token instead of a cookie session